### PR TITLE
Translate README.md to NL and fix badges

### PR DESCRIPTION
**Pull request by Ruben172**

### CODE REVIEW

File: docs/README_CN.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-[![Actions Status](https://github.com/OpenTabletDriver/OpenTabletDriver/workflows/.NET%20Core/badge.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/actions) [![CodeFactor](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/badge/master)](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/overview/master) [![总下载统计](https://img.shields.io/github/downloads/OpenTabletDriver/OpenTabletDriver/total.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/latest)
+[![Actions Status](https://github.com/OpenTabletDriver/OpenTabletDriver/workflows/.NET%20CI/badge.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/actions) [![CodeFactor](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/badge/master)](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/overview/master) [![Total Download Count](https://img.shields.io/github/downloads/OpenTabletDriver/OpenTabletDriver/total.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/latest)
```
Comment (Kuuuube) [inline docs/README_CN.md:1]: Change back the alt text on the download counter. It isn't translated on most of them but it is here so it should be kept.

**Review by Kuuuube — CHANGES_REQUESTED** [review]

File: docs/README_CN.md (hunk 1 above)
Comment (Ruben172) [reply docs/README_CN.md]: thank you kuuube love you <3

File: docs/README_NL.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+[![Actions Status](https://github.com/OpenTabletDriver/OpenTabletDriver/workflows/.NET%20CI/badge.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/actions) [![CodeFactor](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/badge/master)](https://www.codefactor.io/repository/github/OpenTabletDriver/OpenTabletDriver/overview/master) [![Total Download Count](https://img.shields.io/github/downloads/OpenTabletDriver/OpenTabletDriver/total.svg)](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/latest)
+
+# OpenTabletDriver
+
+[English](../README.md) | [한국어](docs/README_KO.md) | [Español](docs/README_ES.md) | [Русский](docs/README_RU.md) | [简体中文](docs/README_CN.md) | [Français](docs/README_FR.md) | [Deutsch](docs/README_DE.md) | [Português-BR](docs/README_PTBR.md) | Nederlands
```
Comment (Kuuuube) [inline docs/README_NL.md:5]:
```suggestion
[English](../README.md) | [한국어](README_KO.md) | [Español](README_ES.md) | [Русский](README_RU.md) | [简体中文](README_CN.md) | [Français](README_FR.md) | [Deutsch](README_DE.md) | [Português-BR](README_PTBR.md) | Nederlands
```
Your links are broken.

**Review by Kuuuube — CHANGES_REQUESTED** [review]

**Comment by jamesbt365** [comment]
good job Ruben amazing commit

**Comment by Ruben172** [comment]
> good job Ruben amazing commit

:3

File: docs/README_NL.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  - Uitvoer modi
+  - Hulpmiddelen
+
+# Bijdragen aan OpenTabletDriver
+
+Bekijk de [issue tracker](https://github.com/OpenTabletDriver/OpenTabletDriver/issues) als u wilt bijdragen aan OpenTabletDriver. Volg de richtlijnen zoals benoemd in onze [richtlijnen voor bijdragen](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/CONTRIBUTING.md) wanneer u een pull request maakt.
+
+[Maak een issue ticket](https://github.com/OpenTabletDriver/OpenTabletDriver/issues/new/choose) als u problemen of suggesties heeft. Vul de template in met relevante informatie. Zowel bug meldingen als tablets om te ondersteunen zijn welkom. Meestal is het ondersteunen van een tablet vrij eenvoudig.
+
+Voor problemen en PRs met betrekking tot de verpakking van OpenTabletDriver, bekijk alstublieft [hier](https://github.com/OpenTabletDriver/OpenTabletDriver.Packaging) de repository.
```
Comment (jamesbt365) [inline docs/README_NL.md:106]: Packaging is now done within this repository.
```suggestion

```

File: docs/README_NL.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+Voor problemen en PRs met betrekking tot de webpagina van OpenTabletDriver, bekijk alstublieft [hier](https://github.com/OpenTabletDriver/OpenTabletDriver.Web) de repository.
```
Comment (jamesbt365) [inline docs/README_NL.md:108]: We have now moved our website to GitHub pages, the current referenced repository is archived and is no longer worked on.
```suggestion
Voor problemen en PRs met betrekking tot de webpagina van OpenTabletDriver, bekijk alstublieft [hier](https://github.com/OpenTabletDriver/opentabletdriver.github.io) de repository.
```

**Review by jamesbt365 — CHANGES_REQUESTED** [review]

**Review by jamesbt365 — APPROVED** [review]
thank

**Review by jamesbt365 — COMMENTED** [review]
Actually I hate to be that guy but the README surrounding building the driver has changed drastically since this was worked on. Could you take a look?

**Comment by Ruben172** [comment]
> Actually I hate to be that guy but the README surrounding building the driver has changed drastically since this was worked on. Could you take a look?

fixed in [9119671](https://github.com/OpenTabletDriver/OpenTabletDriver/pull/2904/commits/9119671f3b9d71d14d062f27d459c51a1a542949)

File: docs/README_NL.md [hunk 5]
```diff
@@ -36,11 +36,11 @@ De benodigdheden om OpenTabletDriver te bouwen zijn hetzelfde voor alle platform
 
 ### Alle platformen
 
-- .NET 6 SDK (kan [hier](https://dotnet.microsoft.com/download/dotnet/6.0) verkregen worden - U heeft de SDK voor uw platform nodig, Linux gebruikers kunnen het via hun package manager installeren indien mogelijk)
+- .NET 6 SDK (kan [hier](https://dotnet.microsoft.com/download/dotnet/6.0) verkregen worden - U heeft de SDK voor uw platform nodig, Linux gebruikers kunnen het via hun package manager installeren waar mogelijk)
```
Comment (jamesbt365) [inline docs/README_NL.md:39]: This should be dotnet 7 sdk, we changed that too.

Comment (Ruben172) [reply docs/README_NL.md:39]: reviewer is fixing that yeah o7

Comment (Ruben172) [reply docs/README_NL.md]: fixed in [5de2056](https://github.com/OpenTabletDriver/OpenTabletDriver/pull/2904/commits/5de20565585b2ae1cfc67e39787f23cb6e6a938e)

**Comment by Earthgames** [comment]
I also check Rubens shit, and it look ok

**Pull request merged**